mintapi/signIn.py

```diff
@@ -12,13 +12,15 @@
 import sys
 import time
 import zipfile
+import itertools
```

Comment:
Is this still needed? I don't see any reference to it.

Reply:
Its for the next at line 149.

Reply:
Got it, thanks!